.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -420,6 +420,13 @@ public List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParseResult
 
             var islunar = val.IsLunar;
             var mod = val.Mod;
+            List<string> list = null;
+
+            // Resolve dates list for date periods
+            if (slot.Type.Equals(Constants.SYS_DATETIME_DATEPERIOD) && val.List != null)
```

Comment:
We only need to add these for BD, no?

Reply:
Yes, the list only added to "BD" cases, for now, this line is able to do this check.